packages/core/content-manager/server/src/controllers/collection-types.ts

```diff
@@ -137,10 +137,10 @@ export default {
       .countRelations({ toOne: false, toMany: true })
       .build();
 
-    const { locale, status } = await getDocumentLocaleAndStatus(query, model);
+    const { locale } = await getDocumentLocaleAndStatus(query, model);
 
     const { results: documents, pagination } = await documentManager.findPage(
-      { ...permissionQuery, populate, locale, status },
+      { ...permissionQuery, populate, locale },
```

Comment:
isn't the status necessary to list published versions?

Reply:
I don't think so, without it I'm seeing it both published and draft/modified entries in the list view

The reason for my change is that requiring `status: published` explicitly here filtered out the draft relations of the entry, wish should be visible in the CM

Reply:
I don't see how the published data version would be listed if we don't specify the status 🤔 if you filter by published entries, you should see the published data on the list view. But after this change would assume users would see the "draft" entry

Reply:
Currently it's not possible to filter by published entries in the list view, though that may be a bug. In the filter select, there's on status option, no publishedAt either.

>Even if marked as "status" = "published" the displayed data would be the draft one

if the status if actually published, that would make no difference right? I think it would only change for a draft/modified entry. And in that case I thought we always wanted to show the more recent draft version, and leave the published one only to the published tab

Reply:
The displayed data would be different, as the draft & published data might differ.
Even if the feature is not enabled, I don't think we should remove that filter though, as I don't think that is the "cause" of the issue, but rather a patch to make it work.

Should we not filter by "status" if D&P is not enabled? We could keep that value as undefined by default